modules/ROOT/pages/connected-apps-developers.adoc

```diff
@@ -112,27 +117,28 @@ The access token can now be used to call Anypoint Platform APIs on the user’s
 
 === Refresh Token
 
-The `refresh_token` grant type is used to grant access to user data when the user isn’t signed in.
+Use the `refresh_token` grant type to grant access to user data when the user isn’t signed in. Refresh tokens expire after 90 days. Access tokens expire after the user’s session expires.
+
 To work with OpenID Connect, apps have to exchange refresh tokens for new access tokens after the initial authorization to continue accessing user data.
-Refresh tokens expire after 90 days.
-Access tokens expire after the user’s session expires.
+
 If the app is using the authorization code grant type, the `offline_access` scope is required to use refresh tokens after the original access tokens expire.
-To request a new access token, you must send a request to the token endpoint using the `refresh_token` grant type.
-When authentication is successful, a refresh token is exchanged for an access token.
+
+To request a new access token, send a request to the token endpoint using the `refresh_token` grant type. When authentication is successful, a refresh token is exchanged for an access token.
 
 === Password
 
-The `password` grant type is used when the application exchanges the user’s username and password for an access token.
-Because the app has to collect the user’s credentials and its limitation with no mechanisms for things like multi-factor authentication, it is not recommended to use this grant type.
+Use the `password` grant type when the application exchanges the user’s username and password for an access token.
+
+Because the app has to collect the user’s credentials and the `password` grant type's limitation in using mechanisms for things like multi-factor authentication, using this grant type isn't recommended.
```

Comment:
Good idea trying to smooth out this wording, but I still find it a little hard to parse, so I think you can take it even further. I'll throw a suggestion in the mix, but you can take it or leave it.

```suggestion
This grant type is not recommended, as it exposes user credentials directly to the connected app. It also precludes the use of additional security measures such as multi-factor authentication.
```